src/App/App.js

```diff
@@ -10,40 +10,73 @@ import Header from '../Header/Header.js';
 import Login from '../Login/Login.js'
 
 // import React, { Component } from 'react';
-import { BrowserRouter, Route, Switch, withRouter } from 'react-router-dom';
+import { BrowserRouter, Switch, Route } from 'react-router-dom';
+
```

Comment:
Don't forget to kill dead code! I believe in you! If you need it back I know that you can rewrite the import without needing to look at it.